apps/docs-snippets/src/guide/cookbook/transferring-assets.test.ts

```diff
@@ -79,7 +69,16 @@ describe(__filename, () => {
     // #endregion transferring-assets-2
   });
 
-  it('should validate that modifying the transaction request will result in another TX ID', async () => {
+  it.skip('should validate that modifying the transaction request will result in another TX ID', async () => {
```

Comment:
The reason we had discussed related to the `MemoryStore` errors , I elaborated more in [this comment](https://github.com/FuelLabs/fuels-ts/issues/2857#issuecomment-2260726481) 

But given https://github.com/FuelLabs/fuel-core/pull/2063 is merged now perhaps it can be unskipped when `fuel-core` v0.33.0 is released and we upgrade to it.